reference/twig_reference.rst

```diff
@@ -614,6 +614,11 @@ Generates the file path inside an ``<a>`` element. If the path is inside
 the kernel root directory, the kernel root directory path is replaced by
 ``kernel.root_dir`` (showing the full path in a tooltip on hover).
```

Comment:
i think even in 4.2 we can already mention `kernel.project_dir` here, that's what will happen looking at `CodeHelper`